Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -64,6 +66,44 @@ public FreezeOnMatchCustomization(Type targetType)
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="FreezeOnMatchCustomization"/> class.
+        /// </summary>
+        /// <param name="parameter">
+        /// The <see cref="ParameterInfo"/> used to freeze specimens decorated with advanced
+        /// attributes (e.g. <see cref="StringLengthAttribute"/>).
+        /// </param>
+        /// <param name="matcher">
+        /// The <see cref="IRequestSpecification"/> used to match the requests
+        /// that will be satisfied by the frozen specimen.
+        /// </param>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="parameter"/> or <paramref name="matcher"/> is null.
+        /// </exception>
+        public FreezeOnMatchCustomization(
+            ParameterInfo parameter,
```

Comment:
Looking on this change, I can predict a question similar to [this](https://github.com/AutoFixture/AutoFixture/pull/627#discussion_r64986467) one I've been asked earlier. Despite it does solve  the original issue, I cannot come up with any *self contained* use case for the new constructor. I tend to create a new internal version of the `FreezeOnMatchCustomization` in the Glue Library and move there all the changes from this class.